app/views/catalog/record/_metadata_panels.html.erb

```diff
@@ -2,6 +2,9 @@
   <%= render "catalog/access_panels/online" %>
   <%= render "catalog/access_panels/course_reserve" %>
   <%= render "catalog/access_panels/location" %>
+  <% if (@document.set_member? && @document.parent_sets.present?) || (@document.is_a_collection_member? && @document.parent_collections.present?) %>
```

Comment:
Is this to try and only render context if `appears_in` or `in_collection` has content?  If so, perhaps we can render both of those into a variable and check that one of them has content.  This way, if the logic w/i one of those partials changes, we don't have to remember to update the logic in this partial to match.